CONTRIBUTING.md

```diff
@@ -0,0 +1,340 @@
+    ``qiskit.foo()`` function which provides similar functionality but with
+    more accurate results and better performance. You should update your calls
+    ``qiskit.bar.foobar()`` calls to ``qiskit.foo()``.
+```
+
+You can also look at other release notes for other examples.
+
+You can use any restructured text feature in them (code sections, tables,
+enumerated lists, bulleted list, etc) to express what is being changed as
+needed. In general you want the release notes to include as much detail as
+needed so that users will understand what has changed, why it changed, and how
+they'll have to update their code.
+
+After you've finished writing your release notes you'll want to add the note
+file to your commit with `git add` and commit them to your PR branch to make
+sure they're included with the code in your PR.
+
+#### Generating the release notes
+
+After release notes have been added if you want to see what the full output of
+the release notes. In general the output from reno that we'll get is a rst
+(ReStructuredText) file that can be compiled by
+[sphinx](https://www.sphinx-doc.org/en/master/). To generate the rst file you
+use the ``reno report`` command. If you want to generate the full Machine Learning release
+notes for all releases (since we started using reno during 0.9) you just run::
+
+    reno report
+
+but you can also use the ``--version`` argument to view a single release (after
+it has been tagged::
+
+    reno report --version 0.5.0
+
+At release time ``reno report`` is used to generate the release notes for the
+release and the output will be submitted as a pull request to the documentation
+repository's [release notes file](
+https://github.com/Qiskit/qiskit/blob/main/docs/release_notes.rst)
+
+#### Building release notes locally
+
+Building The release notes are part of the standard qiskit-machine-learning documentation
+builds. To check what the rendered html output of the release notes will look
+like for the current state of the repo you can run: `tox -edocs` which will
+build all the documentation into `docs/_build/html` and the release notes in
+particular will be located at `docs/_build/html/release_notes.html`
+
+## Installing Qiskit Machine Learning from source
+
+Please see the [Installing Qiskit Machine Learning from
+Source](https://github.com/qiskit-community/qiskit-machine-learning#installation)
+section of the Qiskit documentation.
+
+Note: Machine Learning depends on Terra, and has optional dependence on Aer and IBM Q Provider, so
+these should be installed too. The main branch of Machine Learning is kept working with those other element
+main branches so these should be installed from source too following the instructions at 
+the same location
+
+Machine Learning also has some other optional dependents see 
+[Machine Learning optional installs](https://github.com/qiskit-community/qiskit-machine-learning#optional-installs) for
```

Comment:
```suggestion
Algorithms also has some other optional dependents see 
[Algorithms optional installs](https://github.com/qiskit-community/qiskit-algorithms#optional-installs) for
```
I somehow doubt there are any optional installs - I can't think of any at present. I guess the target here can say there are none and this can be kept